v_3_7_2/master_template.go

```diff
@@ -1443,13 +1443,20 @@ write_files:
       done
 
       # wait for etcd dns (return code 35 is bad certificate which is good enough here)
-      while
-          curl "https://{{ .Cluster.Etcd.Domain }}:{{ .EtcdPort }}" -k 2>/dev/null >/dev/null
-          RET_CODE=$?
-          [ "$RET_CODE" -ne "35" ]
+      # since dns is flapping while we switch back to an elb for etcd we do this 10 times in a row
```

Comment:
I think it would be good to create a TODO issue and track removal of this code https://github.com/giantswarm/fmt/blob/master/go/todo_comments.md

Reply:
I am not sure we should remove it. Any other similar change would cause the same problems again. Also it covers the time until a new master would be registered within the loadbalancer.

Reply:
So the comment should be updated, because we won't be switching back to elb in future versions.